website/docs/faqs/Accounts/slack.md

```diff
@@ -11,8 +11,34 @@ Setting up Slack notifications in dbt Cloud will allow you to receive alerts in
 
 In general, there are two parts to setting up Slack notifications. The first involves setting up the dbt Cloud integration with Slack, while the second involves setting up the notifications themselves.
```

Comment:
~~This section is repeated from the cloud-slack-notifications doc and I wonder if we should use snippets? We might not have time now, but maybe we can create an issue and follow up with it?~~

Actually, maybe we should delete this FAQ?